### PR TITLE
[IMP] base_maintenance: remove unnecessary view

### PR DESCRIPTION
**Pull request by jarroyomorales**
As said in https://github.com/OCA/maintenance/pull/114#discussion_r532464511

@etobella @dalonsod

### CODE REVIEW

**Comment by jarroyomorales** [comment]
Finally I will inherit the original's view and add the name as in newer versions.

File: base_maintenance/views/maintenance_team_views.xml [hunk 1]
```diff
@@ -19,15 +19,14 @@
         </field>
     </record>
 
-    <!-- There is no search view to inherit, so creating it-->
     <record id="maintenance_team_view_search" model="ir.ui.view">
```
Comment (etobella) [inline base_maintenance/views/maintenance_team_views.xml:22]: On 13.0 and 14.0 this view is no longer necessary :smile:

**Review by etobella — APPROVED** [review]

File: base_maintenance/views/maintenance_team_views.xml (hunk 1 above)
Comment (jarroyomorales) [reply base_maintenance/views/maintenance_team_views.xml:22]: For v14 it is removed in the migration, but for 13.0 we may have to leave it like this. Others might be inheriting from it and we can cause them some errors.

**Comment by dalonsod** [comment]
Maybe commit message should be modified, because the view still exists, and, I don't sure but a rebase should be needed to pass codecov

**Comment by jarroyomorales** [comment]
@dalonsod Done!

**Review by dalonsod — APPROVED** [review]
I don't understand codecov errors... as far as I can see, the PR LGTM

**Pull request merged**